examples/SampleApp/package.json

```
@@ -20,6 +21,7 @@
     "@react-navigation/native": "5.9.2",
     "@react-navigation/stack": "5.14.2",
     "@stream-io/flat-list-mvcp": "0.0.9",
+    "patch-package": "^6.4.7",
```

for consistency with everything else
```suggestion
    "patch-package": "6.4.7",
```